docs/design/2020-06-24-placement-rules-in-sql.md

```diff
@@ -20,401 +51,388 @@ The scenarios of defining placement rules in SQL include:
 - Increase the replica count of more important data
 - Separate irrelevant data into different stores to improve availability
 
-## Define placement rules
+These scenarios usually fit into one of the following categories:
 
-There are 3 kinds of operations on the placement:
+1. An optimization or availability use case (such as improving data access locality)
+2. A compliance use case (such as ensuring data resides within a geographic region)
 
-* ADD: Add more replicas for one role.
-* ALTER: Override the replica configuration for one role.
-* DROP: Remove the replica configuration for one role.
+This proposal makes some intentional decisions so that both categories of use-cases can be handled correctly. It improves upon the earlier proposal by reducing the risk of misconfiguration which can affect compliance use cases.
 
-They’re all achieved by executing `ALTER TABLE` statements.
+## Detailed Design
 
-### Add placement rules
+### New Syntax Overview
 
-Adding new replicas can be done by one or more `ADD PLACEMENT POLICY` clauses:
+There are two ways to specify placement rules:
 
-```sql
-ALTER TABLE table_name
-	ADD PLACEMENT POLICY CONSTRAINTS=constraints ROLE=role REPLICAS=replicas,
-	...
-```
+1. By assigning placement directly on a database, table or partition (direct assignment)
+2. By creating a new `PLACEMENT POLICY` and then applying the placement policy to a database, table or partition (placement policy)
+
+Using a `PLACEMENT POLICY` will be recommended for compliance requirements, since it can allow administrators to better keep track of usage. This can be seen as similar to how complex environments will use `ROLES` for management instead of directly assigning privileges to users.
 
-This statement indicates TiDB to add replicas for all data of table `table_name`, including indexes.
+Both syntaxes are considered [`table_option`](https://dev.mysql.com/doc/refman/8.0/en/alter-table.html)s, and available in both `CREATE TABLE` and `ALTER TABLE` contexts.
 
-`ADD PLACEMENT POLICY` is just a part of alter options, just like `ADD COLUMN` or `ADD CONSTRAINT`.
+#### Direct Assignment
 
-To define multiple roles at once, multiple `ADD PLACEMENT POLICY` clauses can appear in a single `ALTER TABLE` statement, even for the same Raft role. For example:
+Creating a new table with directly assigned constraints. The leader is in `us-east-1` region, the followers are in `us-east-1` and `us-east-2`:
 
 ```sql
-ALTER TABLE table_name
-	ADD PLACEMENT POLICY CONSTRAINTS="[+zone=sh]" ROLE=leader REPLICAS=1,
-	ADD PLACEMENT POLICY CONSTRAINTS="[+zone=sh]" ROLE=follower REPLICAS=1
-	ADD PLACEMENT POLICY CONSTRAINTS="[+zone=gz]" ROLE=follower REPLICAS=1;
+CREATE TABLE t1 (
+	id INT NOT NULL PRIMARY KEY,
+	b VARCHAR(100)
+) PRIMARY REGION="us-east-1" REGIONS="us-east-1,us-east-2";
```

Comment:
Yes it will. Unfortunately "region" is the most widely used term for data center geography, so avoiding it's use only complicates our terminology problem.

My proposal is that in documentation we refer to these as "Placement Regions" and update the existing terminology to "Data Regions".